bentoml/_internal/runner/runner_handle/local.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+import typing as t
+from typing import TYPE_CHECKING
+
+from . import RunnerHandle
+
+if TYPE_CHECKING:
+    from ..runner import Runner
+    from ..runnable import Runnable
+
+
+class LocalRunnerRef(RunnerHandle):
```

Comment:
This is just `LocalRunner` right? why call it a `Ref`?

Reply:
LocalRunnerRef
RemoteRunnerClient